### PR TITLE
Update .gitattributes

### PR DESCRIPTION
**Pull request by szepeviktor**
Remove non-existent files, add new development files.

Test command: `git archive HEAD | tar -t --exclude='*/*'`

One more to add: `composer.lock` - the lock file is not used in packages.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -3,24 +3,27 @@
...
 /build/                          export-ignore
 /composer-root-version-checker/  export-ignore
 /docs/                           export-ignore
+/Dockerfile                      export-ignore
 /infection.json.dist             export-ignore
 /fixtures/                       export-ignore
+/.makefile/                      export-ignore
 /Makefile                        export-ignore
-/phpstan.neo.dist                export-ignore
+/phpstan.neon.dist               export-ignore
```
Comment (szepeviktor) [inline .gitattributes:23]: A typo

Comment (theofidry) [reply .gitattributes:23]: Thanks!

Comment (szepeviktor) [reply .gitattributes:23]: Glad to contribute.

**Comment by theofidry** [comment]
Thank you @szepeviktor

**Comment by szepeviktor** [comment]
Actually fixing all these is a solution for **today**.
The Integrity workflow keeps all these fixed at every commit.
https://github.com/szepeviktor/byte-level-care/blob/master/.github/workflows/reusable-integrity.yml#L39
.. maybe this is only valid in my golden cage

**Comment by theofidry** [comment]
I can see a few being helpful, not sure but a few others, but maybe I'll try adding those in the future :)

**Comment by szepeviktor** [comment]
For example the one that looks for gitignored files in the index.
```
$ git ls-files --cached --ignored --exclude-standard
bin/infection.sh
bin/php-scoper
build/.gitkeep
fixtures/set028-symfony/expected-output
fixtures/set030/expected-output
```
Investigating ... 🕵🏻‍♂️

**Pull request merged**